### ISSUE TITLE
Request: Telegram Group or Matrix Chat for Community Discussions

### ISSUE THREAD

**Issue by BakaJzon** [hanydd/BilibiliSponsorBlock#9]
Hi there! I would like to suggest the creation of an official Telegram group or Matrix chat for our community.
This would provide a more dynamic and real-time platform for discussions, support, and collaboration among contributors and users.

**Comment by HanYaodong** [comment]
I've been thinking about this. We do have a QQ group now, but I understand that people not in Mainland China may not use qq at all. Matrix and Telegram are not accessible from Mainland China, so it is a little hard to find a platform that everybody can use comfortably.

Nonetheless, I have been thinking to setup a Telegram group or a Discord channel. Feel free to leave any suggestions!

**Comment by BakaJzon** [comment]
Matrix is a federated chat protocol, meaning anyone can set up their own homeserver and join the Matrix network. This decentralization might make it a viable option even in regions with restrictive internet policies.
Though matrix[.]org is blocked by GFW, there may be still some homeserver(s) available in Mainland China.

Additionally, by using bots, Matrix can be interconnected with Telegram and Discord, allowing seamless communication across platforms. And Matrix is free and open-source.

In short, Matrix is still a great alternative for users inside and outside China.

**Comment by BakaJzon** [comment]
Regarding the consideration of creating a Telegram group, it's important to note that Telegram is often filled with advertisements and scam accounts. To mitigate this, you would need an entry management bot to ensure that most malicious users are kept out. 
Additionally, making the group private could be a viable option to further enhance security and control.

**Comment by hanydd** [comment]
https://t.me/bsbsb_top
